### PR TITLE
feat: add background audio player

### PR DESCRIPTION
**Pull request by kikoqiu**
Add a backgrond audio player.
Able to play all audio files in a folder in a loop or randomly.
Add an icon on the navbar to call it out when it's in background.
background player
No backend  change needed.

### CODE REVIEW

**Comment by HFO4** [comment]
@kikoqiu Great work! I’ll have a try on it first. Thanks a lot!

File: src/component/FileManager/MusicPlayer.js [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+                                </ListItem>
+                            );
+                        })}
+                    </List>
+                    <audio ref={this.myAudioRef}                                 
+                        src={items[currentIndex]?.src}
+                        />
+                    <div style={{"padding-top":8}}></div>
+                    <Grid container spacing={2}>
+                        <Grid item xs>
```
Comment (HFO4) [inline src/component/FileManager/MusicPlayer.js:334]: Add a new style `align-items: center;` for this `Grid` to make the audio control vertical-align.

**Review by HFO4 — CHANGES_REQUESTED** [review]

File: src/component/FileManager/MusicPlayer.js (hunk 1 above)
Comment (kikoqiu) [reply src/component/FileManager/MusicPlayer.js:334]: OK.

Comment (kikoqiu) [reply src/component/FileManager/MusicPlayer.js:337]: Done. 
The slider is not vertically aligned to middle by default. A style patch to the slider is made.

**Comment by HFO4** [comment]
Merged, Thanks!

**Comment by AH-dark** [comment]
This is an excellent idea with a perfact design.

**Pull request merged**